src/main/java/com/diamondfire/helpbot/sys/multiselector/MultiSelector.java

```diff
@@ -11,13 +12,11 @@
 public class MultiSelector {
     
     private final MultiSelectorPage[] pages;
-    private final long channel;
-    private final long user;
+    private final CommandEvent event;
     
-    public MultiSelector(long channel, long user, List<MultiSelectorPage> pages) {
+    public MultiSelector(List<MultiSelectorPage> pages, CommandEvent commandEvent) {
         this.pages = pages.toArray(new MultiSelectorPage[0]);
-        this.channel = channel;
-        this.user = user;
+        this.event = commandEvent;
     }
```

Comment:
No don't hold a command event here, as there is a lot of classes I hold on to in memory then.